automatic-off-session-payment/templates/booking-request-accepted/booking-request-accepted-html.html

```diff
@@ -0,0 +1,92 @@
+
+{{~#*inline "format-money"~}}
+{{money-amount money}} {{money.currency}}
+{{~/inline~}}
+
+{{~#*inline "format-date"~}}
+{{date date format="MMM d, YYYY" tz="Etc/UTC"}}
+{{~/inline~}}
+
+<html>
+  <head>
+    <style type="text/css">
+     table {
+       border-collapse: collapse;
+     }
+     .left {
+       text-align: left;
+     }
+     .right {
+       text-align: right;
+       padding-left: 20px;
+     }
+     .bottom-row > td {
+       padding-bottom: 5px;
+     }
+     .top-row {
+       border-top: 1px solid #CCC;
+     }
+     .top-row > td,
+     .top-row > th
+     {
+       padding-top: 5px;
+     }
+    </style>
+  </head>
+  <body>
+    <h1>Your booking request was accepted!</h1>
+
+    {{#with transaction}}
+    <p>{{provider.display-name}} has accepted your booking request for {{listing.title}} from {{> format-date date=booking.start}} to {{> format-date date=booking.end}}.</p>
+
+    <p>We have charged {{> format-money money=payin-total}} from your credit card. Here's your receipt.</p>
```

Comment:
Is this correct now that the payment happens on a later occasion?

Reply:
Should we state here
- when the payment will happen
- that it happens automatically
- that possibly it may require an action (could leave this out too, can be confusing)

The breakdown we could keep here so that it's already visible to the customer what will be charged once the payment happens.

Reply:
Good point! Reformulated the message to refer to the future, as well as mention that the charge will happen on the card saved for the marketplace.